### PR TITLE
A182 add deputies functionality

### PR DESCRIPTION
**Pull request by pjvv**

### CODE REVIEW

**Comment by pjvv** [comment]
retest this please

File: app/views/deputies/_deputy.html.erb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th>User</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <% deputies["user_deputies"].each do |id, user| %>
+          <tr>
```
Comment (NikolausC) [inline app/views/deputies/_deputy.html.erb]: Needs to check for current user having ownership

File: spec/features/deputiess_spec.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        expect(page).to have_current_path(deputies_path)
+        expect(page).to have_selector('h5', count: @all_deputies.size)
+      end
+
+      it 'will let you create a new deputy' do
+        visit deputies_path
+        expect(page).to have_content('Assign Deputies')
+      end
+
+      it 'shows Delete for stamps that the current user owns' do
```
Comment (NikolausC) [inline spec/features/deputiess_spec.rb]: Should say deputies

File: spec/forms/deputy_form_spec.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      expect(form.id).to be_nil
+    end
+  end
+
+  describe '#save' do
+    context 'validation' do
+      let(:form1) { DeputyForm.new() }
+      let(:form2_1) { DeputyForm.new(user_deputies: 'zs56d87fvbc`/;') }
+      let(:form2_2) { DeputyForm.new(group_deputies: 'zs56d87fvbc`/;') }
+      let(:form3) { DeputyForm.new(user_deputies: 'pirates zombies, clowns', group_deputies: 'zogh') }
```
Comment (NikolausC) [inline spec/forms/deputy_form_spec.rb]: Don't think usernames should have spaces in

File: spec/controllers/deputies_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
```
Comment (NikolausC) [inline spec/controllers/deputies_controller_spec.rb]: Expect vs allow

**Pull request merged**